sdk/communication/communication-short-codes/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
-
+## 1.0.0-beta.2 (2022-02-08)
 ### Features Added
-
+- Updated HTTP client to ``core-rest-pipeline``
 ### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- The new core library no longer exposes the _response property on the operation return types. However, in the operation options, it can take a callback (onResponse) to access the HTTP response. See https://github.com/Azure/autorest.typescript/wiki/%60core-http%60-dependency-migration-to-%60core-client%60-%60core-rest-pipeline%60#change-to-the-_response-property
```

Comment:
this impacts the samples, right? I'm not sure what's the takeaway for me as the end user. I'd reformulate it so that it's clear what's the impact on the end user (if any) or drop it

Reply:
From my understanding, it does impact end users because of  the switch from `RestResponse` to `USProgramBriefResponse`. That being said, I agree with on reformulating the text.